Lib/fontgoggles/font/otfFont.py

```diff
@@ -21,17 +24,17 @@ def _getGlyphDrawing(self, glyphName, colorLayers):
             if self.colorLayers is not None:
                 layers = self.colorLayers.get(glyphName)
                 if layers is not None:
-                    drawingLayers = [(self.ftFont.getOutlinePath(layer.name), layer.colorID)
+                    drawingLayers = [(self._getGlyphOutline(layer.name), layer.colorID)
                                      for layer in layers]
                     return GlyphLayersDrawing(drawingLayers)
             elif self.colorFont is not None:
                 return GlyphCOLRv1Drawing(glyphName, self.colorFont)
 
-        outline = self.ftFont.getOutlinePath(glyphName)
+        outline = self._getGlyphOutline(glyphName)
         return GlyphDrawing(outline)
 
     def varLocationChanged(self, varLocation):
-        self.ftFont.setVarLocation(varLocation if varLocation else {})
+        self.shaper.font.set_variations(varLocation if varLocation else {})
```

Comment:
I think it is this call to `set_variations` that is redundant, and the one in hbShape should stay.